### ISSUE TITLE
Hover, Focus effects for Pagination on background colors

### ISSUE THREAD

**Issue by bewoldt** [uiowa/uiowa#4149]
# Problem
When the pagination is displayed on different backgrounds some of the hover effects are not visible or the color contrast is not enough.

# Examples
https://sandbox.prod.drupal.uiowa.edu/pagination

**Comment by bspeare** [comment]
@bewoldt thanks for finding this!  You could make the updates for this at https://github.com/uiowa/uiowa/blob/main/docroot/themes/custom/uids_base/scss/components/pagination.scss.

**Comment by joewhitsitt** [comment]
Closed with merge of https://github.com/uiowa/uiowa/pull/4166
